emscripten/npm/src/verovio-toolkit.js

```diff
@@ -212,23 +212,28 @@ export class VerovioToolkit {
         if (!options.hasOwnProperty('fontAddCustom')) {
             return options;
         }
-        const filenames = options['fontAddCustom'];
+        const files = options['fontAddCustom'];
         let filesInBase64 = [];
-        // Get all the files and convert them to a base64 string
-        for (let i = 0; i < filenames.length; i++ ) {
+        // Get all the files and convert them to a base64 string if necessary
+        for ( let i = 0; i < files.length; i++ ) {
```

Comment:
```suggestion
        for ( const file of files ) {
```

Using a `for ... of` removes the need to do the next line since you otherwise don't need the iterator `i`.